src/tests/test_analysis_eclipse.py

```diff
@@ -0,0 +1,10 @@
+
+
+def test_run_analysis(setup_eclipse):
+    """
+    Test to run a simple analysis on existing project in Eclipse IDE
+    """
+    eclipse = setup_eclipse
+    eclipse.open_mta_perspective()
+    eclipse.run_simple_analysis(project='acme')
+    assert eclipse.is_analysis_complete()
```

Comment:
Is there a way to assert on story points ?

Reply:
You mean to say that, check story points on the report page? If so, need to discuss that.

Reply:
Yes , we can do that in later PR .